benchmarks/oss_fuzz_benchmark_integration.py

```diff
@@ -0,0 +1,224 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Script for setting up an integration of an OSS-Fuzz benchmark. The script
+will create oss-fuzz.yaml as well as copy the files from OSS-Fuzz to build the
+benchmark."""
+import argparse
+import datetime
+from distutils import spawn
+from distutils import dir_util
+import os
+import logging
+import sys
+import subprocess
+import json
+import bisect
+
+from common import utils
+from common import benchmark_utils
+from common import logs
+from common import new_process
+from common import yaml_utils
+
+
+OSS_FUZZ_DIR = os.path.join(utils.ROOT_DIR, 'third_party', 'oss-fuzz')
+OSS_FUZZ_REPO_PATH = os.path.join(OSS_FUZZ_DIR, 'infra')
+
+
+class GitRepoManager:
+	"""Base repo manager."""
+
+	def __init__(self, repo_dir):
+	    self.repo_dir = repo_dir
+
+	def git(self, cmd):
+	    """Run a git command.
+
+	    Args:
+	      command: The git command as a list to be run.
+
+	    Returns:
+	      stderr, stdout, error code.
+	    """
+	    return new_process.execute(['git'] + cmd,
```

Comment:
super nit: fine to keep as it is, but using `['git', '-C', self_repo_dir] + cmd` might be a tiny bit cleaner than setting cwd